docs/configuration.md

```diff
@@ -19,12 +19,108 @@ The configuration uses the following structure:
 
 ```js
 {
+  threshold: number,
+  floating: Object // optional
 }
 ```
 
 #### Default Configuration Values
 
 ```js
 {
+    threshold: 50,
```

Comment:
you can add typings for the config object and then run documentation tool on it to generate types